File: appointment-service/appointment-service/src/main/java/com/revature/appointmentservice/model/Appointment.java
package com.revature.appointmentservice.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Builder
@Table(name = "appointment")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Appointment {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int id;
	private String reason;
	private String date;
	private String submissionDate;
	private String acceptance;
	private int patientId;
	private String physicianEmail;
}

File: src/app/model/appointment.ts
export class Appointment{
    constructor(
        public id:number,
        public reason:string,
        public submissionDate:Date,
        public date:string,
        public acceptance:string,
        public patientId:number,
        public physicianEmail:string,

    ){
        this.id=id ||0;
        this.reason=reason||"";
        this.acceptance=acceptance||"";
        this.patientId=patientId||0;
        this.physicianEmail=physicianEmail||"";
    }

}